### PR TITLE
Mouse keybinds

### PR DESCRIPTION
**Pull request by Dimava**

### CODE REVIEW

File: src/js/states/keybindings.js [hunk 1]
```diff
@@ -121,9 +121,24 @@ export class KeybindingsState extends TextualGameState {
...
+
+            this.app.settings.updateKeybindingOverride(id, keyCode);
 
+            this.dialogs.closeDialog(dialog);
+            this.updateKeybindings();
+        };
+
+        dialog.inputReciever.backButton.add(() => {});
         this.dialogs.internalShowDialog(dialog);
+        dialog.element.onmousedown = clickListener;
```
Comment (Dimava) [inline src/js/states/keybindings.js:140]: This one is hacky, but Dialog has no global click listener

File: src/js/states/keybindings.js [hunk 2]
```diff
@@ -121,9 +121,24 @@ export class KeybindingsState extends TextualGameState {
             this.updateKeybindings();
         });
 
-        dialog.inputReciever.backButton.add(() => {});
+        const clickListener = event => {
+            console.log(event);
```
Comment (tobspr) [inline src/js/states/keybindings.js:125]: I think you missed a console.log there

File: src/js/game/camera.js [hunk 3]
```diff
@@ -484,6 +485,7 @@ export class Camera extends BasicSerializableObject {
         if (!this.checkPreventDoubleMouse()) {
             return;
         }
+        this.root.app.inputMgr.handleKeyup(event);
```
Comment (tobspr) [inline src/js/game/camera.js:488]: Whats the reason you are manually calling keyup here?

File: src/js/states/keybindings.js (hunk 1 above)
Comment (tobspr) [reply src/js/states/keybindings.js:140]: Could you use addEventListener then?

File: src/js/game/camera.js (hunk 3 above)
Comment (Dimava) [reply src/js/game/camera.js:488]: Because I haven't found how to bind it correctly that time
Fixed.

**Comment by tobspr** [comment]
Thanks a lot!

**Pull request merged**